### DISCUSSION TITLE
Do  I  really need to use the bindings?

### DISCUSSION THREAD

**Discussion by dagadbm** [urql-graphql/urql#1572] [Q&A] [answered]
Hello,

I have probably a dumb question but do I really need to use the bindings to use this?

I have a vue 2 project and I  wanted to use urql as my graphQL client, but there is no support for vue 2.
However, do I really need it ? i dont plan on using graphQL on the components but on the actions of vuex.

**Comment by kitten** [comment, marked as answer]
You don't actually; the bindings are layers on top of the core package which can be used fully standalone https://formidable.com/open-source/urql/docs/basics/core/

**Reply by dagadbm** [reply]
thank  you!
